src/ResourceManager/IotHub/Commands.IotHub.Test/ScenarioTests/IotHubTests.ps1

```diff
@@ -37,8 +37,50 @@ function Test-AzureRmIotHubLifecycle
 	# Create or Update Resource Group
 	$resourceGroup = New-AzureRmResourceGroup -Name $ResourceGroupName -Location $Location 
 
+	Write-Debug " Create new eventHub " 
+	$namespaceName = "IotHubPSEHNamespaceTest"
```

Comment:
This should run against all subs , I see this pattern followed all over the test , this helps us to delete resources if it was there before while running a new test and therefore not leave undeleted resources without usage in the subscription. For test common code , are you suggesting passing this param ?